ckan/controllers/package.py

```diff
@@ -1311,18 +1215,23 @@ def activity(self, id):
         data_dict = {'id': id}
         try:
             c.pkg_dict = get_action('package_show')(context, data_dict)
-            c.pkg = context['package']
-            c.package_activity_stream = get_action(
-                'package_activity_list_html')(
-                context, {'id': c.pkg_dict['id']})
+            package_activity_stream = get_action(
+                'package_activity_list')(
+                context, {'id': id})
```

Comment:
Normally we don't fiddle with context variables and globals because even this style is deprecated, we don't want it to break someone's customized template. However the `package/activity.html` template is fundamentally changed in this PR (`package_activity_stream` changes type from pre-rendered HTML to a list), necessitating changes to a customized template anyway.